packages/elements/src/components/ui/tag/Tag.tsx

```diff
@@ -1,13 +1,14 @@
-import * as React from "react";
+import { getDataProps } from "@stenajs-webui/core";
 import cx from "classnames";
+import * as React from "react";
 import styles from "./Tag.module.css";
-import { getDataProps } from "@stenajs-webui/core";
 
 export interface TagProps {
   variant?: TagVariant;
   label?: string;
   size?: TagSize;
   className?: string;
+  icon?: React.ReactNode;
```

Comment:
`React.ReactNode` is not an icon :) 
For any content, we usually use the name `content`.

But this should probably be `IconInterface` instead if `ReactNode`, which helps ensure that the icon have same look in every application.

Reply:
I double checked with Chris, Tag should not have icon.